docs/shipping/Code/node-js.md

```diff
@@ -930,6 +930,399 @@ helm uninstall logzio-k8s-telemetry
+
+# Install dependencies
+RUN npm install --production
+
+# Copy the application code
+COPY . .
+
+# Expose the application port
+EXPOSE 3000
+
+# Set environment variables for OpenTelemetry
+ENV OTEL_TRACES_SAMPLER=always_on
+ENV OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4317"
+ENV OTEL_RESOURCE_ATTRIBUTES="service.name=nodejs-app"
+
+# Start the application
+CMD ["npm", "start"]
+```
+
+The Dockerfile uses an Alpine-based Node.js image, installs the necessary dependencies, sets the environment variables required for OpenTelemetry configuration, and starts the application.
+
+#### **package.json**
+
+```json
+{
+  "name": "nodejs-app",
+  "version": "1.0.0",
+  "description": "Demo Node.js application with OpenTelemetry instrumentation",
+  "main": "app.js",
+  "scripts": {
+    "start": "node app.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@opentelemetry/api": "^1.9.0",
+    "@opentelemetry/exporter-trace-otlp-grpc": "^0.52.1",
+    "@opentelemetry/sdk-node": "^0.52.1",
+    "express": "^4.21.1"
+  }
+}
+
+```
+
+This file lists the required dependencies, including Express and OpenTelemetry packages for tracing.
+
+#### **task-definition.json**
+
+```json
+{
+  "family": "nodejs-app-task",
+  "networkMode": "awsvpc",
+  "requiresCompatibilities": ["FARGATE"],
+  "cpu": "256",
+  "memory": "512",
+  "executionRoleArn": "arn:aws:iam::<aws_account_id>:role/ecsTaskExecutionRole",
+  "containerDefinitions": [
+    {
+      "name": "nodejs-app",
+      "image": "<aws_account_id>.dkr.ecr.<region>.amazonaws.com/nodejs-app:latest",
+      "cpu": 128,
+      "portMappings": [
+        {
+          "containerPort": 3000,
+          "protocol": "tcp"
+        }
+      ],
+      "essential": true,
+      "environment": [],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/ecs/nodejs-app",
+          "awslogs-region": "<aws-region>",
+          "awslogs-stream-prefix": "ecs"
+        }
+      }
+    },
+    {
+      "name": "otel-collector",
+      "image": "<aws_account_id>.dkr.ecr.<aws-region>.amazonaws.com/otel-collector:latest",
+      "cpu": 128,
+      "essential": false,
+      "command": ["--config=/etc/collector-config.yaml"],
+      "environment": [
+        {
+          "name": "LOGZIO_TRACING_TOKEN",
+          "value": "<logzio_tracing_token>"
+        },
+        {
+          "name": "LOGZIO_REGION",
+          "value": "<logzio_region>"
+        }
+      ],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/ecs/otel-collector",
+          "awslogs-region": "<aws-region>",
+          "awslogs-stream-prefix": "ecs"
+        }
+      }
+    }
+  ]
+}
+```
+
+This task definition includes both the Node.js application container and the OpenTelemetry Collector container, defining their configurations and log groups.
+
+### **Step-by-Step Instructions**
```

Comment:
Same comment as on the java - In my opinion it would be most beneficial for users to have the files (if relevant) under the relevant step.
And if it's an example app - we can place in the examples repo and add a note with a link to it